src/Coriolis/Coriolis.jl

```diff
@@ -25,14 +25,21 @@ Abstract supertype for parameters related to background rotation rates.
 """
 abstract type AbstractRotation end
 
+abstract type HydrostaticCoriolis end
+
+abstract type NonhydrostaticCoriolis end
+
 const face = Face()
 const center = Center()
 
+const HydrostaticSphericalCoriolis{S, FT} = SphericalCoriolis{HydrostaticCoriolis, S, FT}
+const NonhydrostaticSphericalCoriolis{S, FT} =  SphericalCoriolis{NonhydrostaticCoriolis, S, FT}
```

Comment:
I think this should go in the `spherical_coriolis.jl` file

Reply:
I just moved those as well.